test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Lastly, as you rightly wrote here in this file, this file should be ignored by Git. Hence it shouldn't even be on GitHub so since it is here as part of your files that means it wasn't ignored. After you add this file to your .gitignore file, it shouldn't be pushed to GitHub. 👍 So kindly check and fix this 😉

 

- [ ] You can read more about how `.gitignore` works [here](https://www.bmc.com/blogs/gitignore/).